cloudconfig/cloudinit/cloudinit_centos.go

```diff
@@ -33,10 +33,9 @@ type centOSHelper struct {
 func (helper centOSHelper) getRequiredPackages() []string {
 	return []string{
 		"curl",
-		"bridge-utils",
```

Comment:
@hpidcock I *think* this might be required for creating e/n/i based bridges for LXD/KVM.